portfolio/src/main/webapp/script.js

```diff
@@ -24,22 +28,12 @@ function generateRandomGreeting(){
     return greeting;
 }
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = generateRandomGreeting();
-}
-
 /**
  * Change the innerHTML to a greeting
  * every time use loads or refreshes
  */
 function onLoadWindow(){
-    document.getElementById("welcome-note").innerHTML = generateRandomGreeting().concat(" My name is Roland");
+    document.getElementById("welcome-note").innerHTML = generateRandomGreeting().concat(" My name is ", getUserName());
```

Comment:
A way to make this a little more readable:

```
document.getElementById("welcome-note").innerHTML = `${generateRandomGreeting()} My name is ${getUserName()}`;
```

Reply:
> A way to make this a little more readable:
> 
> ```
> document.getElementById("welcome-note").innerHTML = `${generateRandomGreeting()} My name is ${getUserName()}`;
> ```

This is a helpful shortcut on how to concatenate. I will start applying it ASAP